Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -108,7 +108,6 @@ struct CacheSystem: Sendable {
     static let defaultParalellNumber = 8
     private let pinsStore: PinsStore
     private let outputDirectory: URL
-    private let storage: (any CacheStorage)?
```

Comment:
Due to the introduction of `.storages` cache mode, storages may be different on when saving caches and when restoring caches. So removed the property and passes by arguments instead.